.github/workflows/dynamoAllNet6.0_build.yml

```diff
@@ -18,12 +18,12 @@ jobs:
       run: echo "::remove-matcher owner=csc::"
 
     - name: Install dependencies for windows runtime
-      run: dotnet restore $Env:GITHUB_WORKSPACE\Dynamo\src\Dynamo.All.Net6.sln -p:Platform=NET60_Windows --runtime=win10-x64
+      run: dotnet restore $Env:GITHUB_WORKSPACE\Dynamo\src\Dynamo.All.sln -p:Platform=NET60_Windows --runtime=win10-x64
     - name: Build Dynamo with MSBuild for Windows
       run: |
         echo "***Continue with the build, Good luck developer!***"
         cd "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\MSBuild\Current\Bin\"
-           .\MSBuild.exe $Env:GITHUB_WORKSPACE\Dynamo\src\Dynamo.All.Net6.sln /p:Configuration=Release  /p:Platform=NET60_Windows
+           .\MSBuild.exe $Env:GITHUB_WORKSPACE\Dynamo\src\Dynamo.All.sln /p:Configuration=Release  /p:Platform=NET60_Windows
```

Comment:
Do you want to rename this yml file too to remove the "net6.0" from the name?

Reply:
Well the yaml is suggestive that we are building based on net6 (the dotnet framework)
Maybe that is somewhat helpful
if not I can rename it...
I can imagine another yaml for net8 (dynamoAllNet8.0_build)